fleetspeak/src/client/channel/channel.go

```diff
@@ -84,10 +86,25 @@ type Channel struct {
 
 	magicRead chan bool // signals if the first magic number read succeeds or fails
 	inProcess sync.WaitGroup
+
+	startupDataOut *fcpb.StartupData        // Startup data to send to the other process.
+	StartupDataIn  <-chan *fcpb.StartupData // Startup data sent by the other process.
```

Comment:
nit: suggest "received from other process" - matches above and is slightly more precise